style.css

```diff
@@ -60,328 +67,425 @@ body {
+.skill-detail li img {
+  padding: 12px;
+}
+
+.skill-detail li h3 {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: 253858;
+}
+
+/* Contact form  */
+
+footer {
+  background-color: #fff;
+}
+
+#contact-form {
+  background-image: url(/images/contact-form.png);
+  background-repeat: no-repeat;
+  background-size: 60% 90%;
+  background-position: top right;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+}
+
+#contact-form h1 {
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  padding-top: 103px;
+  line-height: 52px;
+  color: #fff;
+}
+
+#contact-form p {
+  text-align: center;
+  color: #fff;
+  font-weight: 400;
+  line-height: 28px;
+  margin: 12px 24px;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 24px;
+  gap: 20px;
+}
+
+.form-container input[type=text],
+.form-container input[type=email] {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+  height: 48px;
+}
+
+.form-container textarea {
+  padding: 15px 38px 15px 16px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 170px;
+  resize: none;
+  font-size: 15px;
+}
+
+::placeholder {
+  color: #b3bac5;
+}
+
+.submit-btn {
+  color: #6070ff;
+  margin-bottom: 45px;
+}
+
+/* Desktop version */
+
+@media all and (min-width: 768px) {
+  /* Header */
+  #header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 0 143px;
+    justify-content: space-between;
+    height: 72px;
+    align-items: center;
   }
 
-  .btn {
-    text-align: center;
-    letter-spacing: 0.03em;
-    font-size: 17px;
+  #navbar .web {
+    display: block;
   }
 
-  /* About myself section  */
-  #about-myself {
-    background: #fff;
-    border-bottom-left-radius: 0;
-    border-top-right-radius: 100px;
-    height: auto;
+  #navbar .mobile {
+    display: none;
   }
 
-  #about-myself h1 {
-    margin-top: 114px;
+  .presentation {
+    background-image: url(images/header.png);
+    justify-content: center;
+    margin-top: 72px;
+    padding: 0 260px;
   }
 
-  #resume {
-    width: 80%;
+  .presentation h1 {
+    font-size: 48px;
+    line-height: 60px;
   }
 
-  #resume button {
-    color: #396df2;
+  .presentation p {
+    font-size: 20px;
+    line-height: 28px;
   }
 
-  .skill-set {
-    width: 80%;
-    display: flex;
-    flex-flow: column wrap;
+  .card {
+    flex-direction: row;
+    gap: 46px;
+    justify-content: center;
   }
 
-  .skill-set ul li {
-    display: flex;
-    background-color: #f7f7f9;
+  .card img.snapshot {
     border-radius: 8px;
-    align-items: center;
-    height: 67;
+    justify-content: center;
+    align-self: center;
+    width: 100%;
+    min-width: 50%;
   }
 
-  #language {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    height: 72px;
+  .card .right {
+    order: -1;
   }
```

Comment:
- [ ] And here as well, I think you can increase the padding-left, this is going to help your portfolio look more like the Figma template. 🤗

` .card . right { padding-left: 20px; }`